ci-operator/step-registry/upi/deprovision/powervs/cluster/upi-deprovision-powervs-cluster-ref.yaml

```diff
@@ -0,0 +1,23 @@
+ref:
+  as: upi-deprovision-powervs-cluster
+  from: upi-installer
+  commands: upi-deprovision-powervs-cluster-commands.sh
+  grace_period: 10m
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  dependencies:
+  - name: "release:latest"
+    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+  env:
+  - name: RESOURCE_GROUP
+    default: ""
+  - name: BASE_DOMAIN
+    default: ""
+  - name: JQ_VERSION
+    default: ""
+  - name: UPI_AUTOMATION_VERSION
```

Comment:
may be worth adding `documentation: "the tag from https://github.com/ocp-power-automation/openshift-install-power/releases"`